### ISSUE TITLE
Removal of ores doesnt fully work

### ISSUE THREAD

**Issue by Hojosa** [KubeJS-Mods/KubeJS#846]
### Minecraft Version

1.20.1

### KubeJS Version

2001.6.5-build.7

### Rhino Version

2001.2.2-build.6

### Architectury Version

9.2.14

### Forge/Fabric Version

NeoForge 47.1.105

### Describe your issue

Using event.removeOres doesnt remove all ores listed, similar to #775

Script:
`WorldgenEvents.remove(event => {
  event.removeOres(props => {
    props.worldgenLayer = 'underground_ores';
    props.blocks = [
      'immersiveengineering:ore_lead',
      'immersiveengineering:deepslate_ore_lead',
      'immersiveengineering:ore_nickel',
      'immersiveengineering:deepslate_ore_nickel',
      'immersiveengineering:ore_silver',
      'immersiveengineering:deepslate_ore_silver',
      'mekanism:tin_ore',
      'mekanism:deepslate_tin_ore'
    ]
  })
})`

will only remove immersiveengineering:ore_silver. additionally, it seems to always be one from the list and the rest is not removed. i got another test setup (with more mods) where it only removed the nickel ore. 
I also checked the source code of ImmersiveEngineering, and all Ores are placed in underground_ores, as far as i can tell. 

Mods used:
ImmersiveEngineering-1.20.1-10.1.0-171
Mekanism-1.20.1-10.4.8.43
theoneprobe-1.20.1-10.0.2

**Comment by desagas** [comment]
Did you ever figure this out?

I have `WorldgenEvents` in my startup_scripts folder, but, every time I try to load it, I get this error in 1.21.1:
`[ERROR] ! worldgen.js#1: dev.latvian.mods.rhino.EcmaError: ReferenceError: "WorldgenEvents" is not defined.`

Am I missing something?

**Comment by Hojosa** [comment]
nope, in your case, the 1.21 version most likely doesnt have the WorldgenEvents implemented yet
